core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -174,6 +174,35 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
     getOption(key).getOrElse(defaultValue)
   }
 
+  /** Get a time parameter as seconds; throws a NoSuchElementException if it's not set. If no 
+    * suffix is provided then seconds are assumed. 
+    */
```

Comment:
style: in Spark we use java docs not scala docs. See other methods in this file for reference